### PR TITLE
Add postmortems section to master branch 

### PR DESCRIPTION
**Pull request by spitalnyd**
The postmortems section of the code of conduct is complete, all checks pass in Travis with a green checkmark present. This pull-request would merger the team4-postmortems branch into the master branch. Please comment your suggestions, concerns, and feedback would be much appreciated.

### CODE REVIEW

File: conduct.md [hunk 1]
```diff
@@ -13,7 +13,29 @@
...
+and what failed and needs improvement. These discussions also provide great
+insight in how to improve in future projects.
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * Create an agenda of things that need to be reviewed and fixed
+    * Participate in the discussions in a respectful and courteous manner
+    * Provide constructive criticism and propose solutions for issues faced
+    * Receive feedback and work on improving in the future, criticism
+    is not personal
```
Comment (corlettim) [inline conduct.md]: I would maybe clarify or reword this statement. The second half does not relate to receiving feedback.

Comment (noorbuchi) [reply conduct.md]: I tried to clarify these lines. let me know it this is better. Thanks!

**Review by noorbuchi — APPROVED** [review]

**Review by gkapfham — CHANGES_REQUESTED** [review]
Hello @spitalnyd and @noorbuchi, thanks for working on this PR!

After reviewing this PR, I have several suggestions for small improvements:

- Can you clarity what is meant by "future work"?
- There is at least one bullet point in which you do not write "constructive criticism". Even though it is somewhat repetitive in nature, I suggest that you always use this phrasing and avoid only writing "criticism" so that it is clear what type of feedback we will value.
- Is every member of the team responsible for creating the agenda? Or, is that the task of a specific member of the team who starts one of the postmortems?

**Comment by noorbuchi** [comment]
Hello @gkapfham , thanks for the feedback! I included clarifications in my recent commit that answer many of the questions in you change request. Please let me know if additional changes are needed. Thanks!

**Review by enpuyou — APPROVED** [review]

**Review by corlettim — APPROVED** [review]
Looks good now!

**Review by gkapfham — APPROVED** [review]
Hi @spitalnyd, I'm glad to approve this pull request. With that said, I suggest that we consider whether or not there is going to be an emoji for "Acceptable" and "Unacceptable" or if we are still going to use the letters. What does everything think? With that said, I don't think that issue is so pressing that we can't go ahead and merge this PR and this address it at a later date. Thanks!

**Comment by noorbuchi** [comment]
Hi @gkapfham , the usage of emojis is exclusive to the assessment repository. The standard present in the markdown repository doesn't include the usage of emojis in the conduct files. Thank you!

**Comment by carmop** [comment]
To standardize the writing between all files we removed the italicization and bold from the title of the subsections.

**Review by ibarrezueta — APPROVED** [review]

**Pull request merged**